packages/cxl-jwplayer/src/components/cxl-jwplayer.js

```diff
@@ -1,15 +1,10 @@
 import { LitElement, html, customElement, property } from 'lit-element';
 import '@conversionxl/cxl-lumo-styles';
-// import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
-import cxlJwplayerStyles from '../styles/cxl-jwplayer-css.js';
-// import cxlJwplayerGlobalStyles from '../styles/global/cxl-jwplayer-css.js';
+import { registerGlobalStyles } from '@conversionxl/cxl-lumo-styles/src/utils';
+import cxlJwplayerGlobalStyles from '../styles/global/cxl-jwplayer-css.js';
```

Comment:
the `global` in the path caused my build to fail. After removing it it worked.

Reply:
`yarn build` failed? or `yarn storybook`?

Reply:
both failed.

Reply:
hum, I think we actually have a little bug with the building system here, @lkraav 
I remember now I had to manually create the global folder for the build to work. Should we open a separate issue to deal with it, or is it a know problem with a known workaround?

Reply:
Yes, you need to create JS output folders manually for now, with a corresponding `.gitignore` file (see existing). Unless it's really simple, not worth fixing for automation.